dockstore-webservice/src/main/resources/requirements/1.13.0/requirements3.txt

```diff
@@ -0,0 +1,31 @@
+bagit==1.8.1
```

Comment:
Just curious, how do you create this file? Some Python thing I'm unaware of, I'm sure.

Reply:
I simply copied requirements2.txt and updated the package versions, and tested the install on my local machine, which succeeded. I believe you can find cwltool package dependencies as explained [here](https://stackoverflow.com/questions/9232568/identifying-the-dependency-relationship-for-python-packages-installed-with-pip), which I did; it showed the following additional required packages which are not listed in the requirements3.txt file:
charset-normalizer
pydot
setuptools

Perhaps these should be added?

Reply:
>I simply copied requirements2.txt and updated the package versions, and tested the install on my local machine, which succeeded. I believe you can find cwltool package dependencies as explained [here](https://stackoverflow.com/questions/9232568/identifying-the-dependency-relationship-for-python-packages-installed-with-pip), which I did; it showed the following additional required packages which are not listed in the requirements3.txt file:
charset-normalizer
pydot
setuptools
>
> Perhaps these should be added?

I don't know, going to defer to @Richard-Hansen and/or @david4096 on Python best practices.

Reply:
I think you may be right about using the CWLTool GitHub page requirements.txt file

Reply:
I'd seen that one, but it didn't include cwltool itself, so I was confused. I guess it's the requirements needed by cwltool, but not including cwltool proper?

Reply:
@wshands I'd suggest adding the CWLTool's requirements.txt from the desired release to our current requirements file (for any missing dependencies or mismatched versions).

Reply:
Added missing packages from CWLTool's requirements.txt from the desired release and added charset-normalizer
pydot
setuptools
so all packages and versions are specified.